Cargo.toml

```diff
@@ -79,6 +80,7 @@ default = [
     "noise",
     "rustls_tls",
     "sanitize",
+    "sound_len",
```

Comment:
no

Reply:
tgstation sound maintainer wants this feature so it makes sense to me